resources_pages/install_ds_stack_ubuntu.md

```diff
@@ -25,7 +25,7 @@ subtitle: MDS software stack install instructions for Ubuntu 2022/23
 - [LaTeX](#latex){:target="_self"}
 - [PostgreSQL](#postgresql){:target="_self"}
 - [Docker](#docker){:target="_self"}
-- [Quarto](#quarto){:target="_self"}
+- [Quarto CLI](#quarto-cli){:target="_self"}
```

Comment:
Don't forget to change the heading in this file also. I kept it as "Quarto for the ToC to work" https://github.com/UBC-MDS/UBC-MDS.github.io/blob/e1ea50a827ae16e032dcbf6e2e913f78fe046f5d/resources_pages/install_ds_stack_ubuntu.md?plain=1#L599

Reply:
I added this myself just now